api/TextTrackList.json

```diff
@@ -20,7 +20,7 @@
             "version_added": null
           },
           "ie": {
-            "version_added": null
+            "version_added": "10"
```

Comment:
`TextTrackList` is used for https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/textTracks which is missing the IE data. Can you update that here too, to ensure they're in sync? It would be totally possible for the `TextTrackList` interface object to be missing, so if `textTracks` is supported earlier than this, we'd need to investigate. (IE9 seems too early for that to be possible, though.)

Reply:
Apparently the page just isn't updating with the most recent information, but going to `wiki.***` shows the up-to-date BCD.

Reply:
Ah, so that entry was updated in https://github.com/mdn/browser-compat-data/pull/7246, now says IE10.